pkg/services/ngalert/api/tooling/definitions/alertmanager_test.go

```diff
@@ -130,7 +130,6 @@ alertmanager_config: |
 			// Override the map[string]any field for test simplicity.
 			// It's tested in Test_GettableUserConfigRoundtrip.
 			out.amSimple = nil
-			require.Equal(t, tc.output, out)
```

Comment:
Possibly removing this wasn't intentional?

Reply:
Indeed 😅